boards/msba2/include/board.h

```diff
@@ -55,6 +55,17 @@ extern "C" {
 #define XTIMER_SHOOT_EARLY  3
 /** @} */
 
+/**
+ * @name    Configure conntect CC1101 (radio) device)
+ * @{
+ */
+#define CC110X_SPI          0
+#define CC110X_CS           53
```

Comment:
Those values are weird, is it possible to find something with GPIO_PIN(PORT_NUM, PIN_NUM) ? I know it's out of the scope of this PR, I'm just commenting

Reply:
In `cpu/lpc2387/include/periph_cpu.h:57` I found this
```C
#define GPIO_PIN(port, pin) (port*32 + pin)
```

So I guess it should be like this
```
CS   = 53 = GPIO_PIN(PORT_B, 21)
GDO0 = 27 = GPIO_PIN(PORT_A, 27)
GDO1 = 55 = GPIO_PIN(PORT_B, 23)
GDO2 = 28 = GPIO_PIN(PORT_A, 28)
```

I will check this tomorrow on the hardware. Would be nice to have no magic numbers in the code.

Reply:
No need to test, this will obviously yield the same value and thus has to be correct.

Reply:
I think this is better:

```
CS   = 53 = GPIO_PIN(1, 21)
GDO0 = 27 = GPIO_PIN(0, 27)
GDO1 = 55 = GPIO_PIN(1, 23)
GDO2 = 28 = GPIO_PIN(0, 28)
```

This would be consistent with the rest of the MSBA2 `board.h` and the pin labels at the board diagram.

![board diagram](https://raw.githubusercontent.com/wiki/RIOT-OS/RIOT/images/MSBA2_layout.png)